keys.go

```diff
@@ -135,6 +136,19 @@ func dataKeyToCid(s string) (cid.Cid, error) {
 	return cid.Decode(s[1 : len(s)-len(dataSuffixKey.String())])
 }
 
+func tagKeyToCid(k datastore.Key) (cid.Cid, error) {
+	s := k.String()
+	min := 4
```

Comment:
would having `min` as a global variable be useful?

Reply:
It could, but it's only used in this one function.